lib/Util/DefaultLogger.php

```diff
@@ -8,15 +8,22 @@
  */
 class DefaultLogger implements LoggerInterface
 {
+    /** @var int */
     public $messageType = 0;
 
+    /** @var null|string */
     public $destination;
 
     public function error($message, array $context = [])
     {
         if (\count($context) > 0) {
             throw new \Stripe\Exception\BadMethodCallException('DefaultLogger does not currently implement context. Please implement if you need it.');
         }
-        \error_log($message, $this->messageType, $this->destination);
+
+        if (null === $this->destination) {
+            \error_log($message, $this->messageType);
+        } else {
+            \error_log($message, $this->messageType, $this->destination);
```

Comment:
Likewise, the third argument to [`error_log()`](https://www.php.net/manual/en/function.error-log.php) is optional, but should be a string when present.